diagnostics-app/src/webcomponents/hello-world/hello-world.element.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2021 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+@customElement('hello-world')
+export class HelloWorldElement extends LitElement {
+  render() {
+    return html` <h1>Hello World from LitElement!</h1> `;
+  }
```

Comment:
What is the purpose of this file? It seems like you are testing LitElement? Let's not add unused code.

Reply:
Yes, I included it as a dummy component for future references. Removed it. 👍🏻 I'll make sure to not push unused code from next time.